libsel4/sel4_arch_include/aarch32/sel4/sel4_arch/constants.h

```diff
@@ -71,6 +71,7 @@ typedef enum {
     SEL4_FORCE_LONG_ENUM(seL4_VCPUFault_Msg),
 } seL4_VCPUFault_Msg;
 
+/* If the parity of seL4_VCPUReg_Num changes, the padding inside the vcpu struct must also change */
```

Comment:
With the compile_assert, this comment is unnecessary.

Reply:
I'd like to leave it in. The assert will fail, but it was still a discussion every single time to find out why the verification failed for this when it was violated (also the assert will not fail in the verified config, because they are compiled out there).